packages/utils/pack-up/src/node/core/logger.ts

```diff
@@ -73,5 +74,13 @@ export const createLogger = (options: LoggerOptions = {}): Logger => {
 
       console.error(chalk.red(`[ERROR] `), ...args);
     },
+
+    success(...args) {
+      if (silent) {
+        return;
+      }
+
+      console.info(chalk.green(`[SUCCESS] `), ...args);
```

Comment:
Hijacking this for something I missed in a previous PR. It feels weird that the logger methods with matching names to console don't actually use those methods. eg, logger.log -> console.info,  logger.debug -> console.log